apps/condo/domains/banking/schema/ImportBankTransactionsService.js

```diff
@@ -0,0 +1,201 @@
+const { Organization } = require('@condo/domains/organization/utils/serverSchema')
+
+/**
+ * List of possible errors, that this custom schema can throw
+ * They will be rendered in documentation section in GraphiQL for this custom schema
+ */
+const errors = {
+    INVALID_FILE_FORMAT: {
+        mutation: 'importBankTransactions',
+        variable: ['data', 'file'],
+        code: BAD_USER_INPUT,
+        type: 'INVALID_FILE_FORMAT',
+        message: 'Cannot parse uploaded file',
+        messageForUser: 'api.banking.importBankTransactions.INVALID_FILE_FORMAT',
+    },
+    CANNOT_FIND_BANK_INTEGRATION: {
+        mutation: 'importBankTransactions',
+        code: INTERNAL_ERROR,
+        type: 'CANNOT_FIND_BANK_INTEGRATION',
+        message: 'Cannot find BankIntegration with id "61e3d767-bd62-40e3-a503-f885b242d262" corresponding to import from file in "1CClientBankExchange" format',
+    },
+    SOMETHING_WENT_WRONG: {
+        mutation: 'importBankTransactions',
+        code: INTERNAL_ERROR,
+        type: INTERNAL_ERROR,
+        message: 'Error during import',
+        messageForUser: 'api.user.importBankTransactions.SOMETHING_WENT_WRONG',
+    },
+    ANOTHER_INTEGRATION_IS_USED: {
+        mutation: 'importBankTransactions',
+        variable: ['data', 'file'],
+        code: BAD_USER_INPUT,
+        type: 'ANOTHER_INTEGRATION_IS_USED',
+        message: 'Another integration is used for this bank account, that fetches transactions in a different way. You cannot import transactions from file in this case',
+        messageForUser: 'api.banking.importBankTransactions.ANOTHER_INTEGRATION_IS_USED',
+    },
+    BANK_ACCOUNT_HAS_NO_INTEGRATION: {
+        mutation: 'importBankTransactions',
+        variable: ['data', 'file'],
+        code: BAD_USER_INPUT,
+        type: 'BANK_ACCOUNT_HAS_NO_INTEGRATION',
+        message: 'For this import operation was found bank account with id "${id}" and it does not have integration. It means that this bank account has been initially used for something different or something goes wrong. In this case it cannot be used to fetch data from file',
+        messageForUser: 'api.banking.importBankTransactions.BANK_ACCOUNT_HAS_NO_INTEGRATION',
+    },
+}
+
+const ImportBankTransactionsService = new GQLCustomSchema('ImportBankTransactionsService', {
+    doc: {
+        summary: 'Imports bank transactions from file in 1CClientBankExchange format of version 1.03',
+        errors,
+    },
+    types: [
+        {
+            access: true,
+            type: 'input ImportBankTransactionsInput { dv: Int!, sender: JSON!, file: Upload!, organizationId: ID! }',
+        },
+        {
+            access: true,
+            type: 'type ImportBankTransactionsOutput { bankAccount: BankAccount }',
+        },
+    ],
+    
+    mutations: [
+        {
+            access: access.canImportBankTransactions,
+            schema: 'importBankTransactions(data: ImportBankTransactionsInput!): ImportBankTransactionsOutput',
```

Comment:
If it's support just a one format the format should be a part of the name

Reply:
Why you want to create a custom mutation? You can create a model and run parsing in task.

It doesn't looks like 300 ms endpoint.

Reply:
Yes, this way it's quicker to implement it.
I've planned to execute it in a task for next iteration.